app/dashboard/templates/shared/nav_auth.html

```diff
@@ -33,7 +33,7 @@
           {% trans "Send Tip" %}
         </a>
         <a class="dropdown-item" href="{% url "kudos_marketplace" %}">
-          <i class="fas fa-magic"></i>
+          <i class="fas"><img src="{% static "v2/images/kudos/IconKudos.svg" %}"></i>
```

Comment:
@usmanmuhd could we remove the fas class too ?

Reply:
Here as well it is required to keep the icon aligned in the same way as the other icons.

Reply:
Ah ok 👍